projects/Cura.cmake

```diff
@@ -0,0 +1,7 @@
+ExternalProject_Add(Cura
+    GIT_REPOSITORY https://github.com/ultimaker/Cura
+    GIT_TAG origin/${TAG_OR_BRANCH}
+    CMAKE_ARGS -DCMAKE_INSTALL_PREFIX=${EXTERNALPROJECT_INSTALL_PREFIX} -DCMAKE_PREFIX_PATH=${CMAKE_INSTALL_PREFIX} -DURANIUM_SCRIPTS_DIR= -DCURA_VERSION=${CURA_VERSION}
```

Comment:
Should `-DURANIUM_SCRIPTS_DIR` be empty? I suspect this might not work with the generating of .mo files for translations.